pilot/pkg/config/cloudfoundry/fakes/copilot_client.go

```diff
@@ -249,4 +248,4 @@ func (fake *CopilotClient) recordInvocation(key string, args []interface{}) {
 	fake.invocations[key] = append(fake.invocations[key], args)
 }
 
-var _ monitor.CopilotClient = new(CopilotClient)
+var _ cloudfoundry.CopilotClient = new(CopilotClient)
```

Comment:
can you postpone this move for a week until we do the last merge of release-1.0 to master?

Reply:
if you want to wait until 1.0 is merged into master next week that is totally fine - lets just leave this PR open until then (I will make sure to rebase after the final merge).